### PR TITLE
💚 (repo): Fixing gitignores and yarn issues

### PR DESCRIPTION
**Pull request by arneschreuder**
This PR attempts to fix the CI/CD pipeline

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
-yarn.lock 
+yarn.lock
```
Comment (arulagrawal) [inline .gitignore:3]: I think yarn.lock should _not_ be in the .gitignore at all. [Official yarn Q&A](https://yarnpkg.com/getting-started/qa/#should-lockfiles-be-committed-to-the-repository)

Comment (arneschreuder) [reply .gitignore:3]: Agreed, sorry, I missed this. But I think the issue might be related to something else... Will provide feedback tonight

**Comment by arneschreuder** [comment]
Ok, so this commit [4ab67b8a5943dd626640079d1e177b695adac768 ](https://github.com/COS-301/graduates/commit/4ab67b8a5943dd626640079d1e177b695adac768) is where things break. I will resolve it now.

**Comment by arneschreuder** [comment]
> Ok, so this commit [4ab67b8a5943dd626640079d1e177b695adac768 ](https://github.com/COS-301/graduates/commit/4ab67b8a5943dd626640079d1e177b695adac768) is where things break. I will resolve it now.

More accurately, here: [08fa38ba87d48ae8919ce51a16e33607c36eac84](https://github.com/COS-301/graduates/commit/08fa38ba87d48ae8919ce51a16e33607c36eac84)

**Comment by Alexander-Muendesi** [comment]
@arneschreuder is there anyway we could assist you in fixing the current problem with the linter? Things to look for or anything which you might be trying

**Comment by arneschreuder** [comment]
To be honest, Ive been trying to fix the repo since yesterday. Still no luck. Cant seem to narrow down exactly what causes the issue. But I should have it fixed this weekend

**Comment by Alexander-Muendesi** [comment]
could the reinstallation of node modules in this PR https://github.com/COS-301/graduates/pull/458/commits maybe have something to do with, whether something did not install correctly or a version number changed which is messing things up. I spent some time looking at the code and could not find much reading on similar errors which were not necessarily specific to eslint

**Comment by arneschreuder** [comment]
Closing this in favour of another PR.

**Comment by arneschreuder** [comment]
I found the issue. Now just need to fix it. Busy with that now. This took incredibly long, but no harm done. Just a blocker. But thats part of the process. Will let you know when the fix is ready

**Pull request closed without merging**